host_vars/hosts.yml

```diff
@@ -0,0 +1,22 @@
+---
+raspberry:
+  hosts:
+    rasp3:
+      ansible_host: 192.168.0.25
+      ansible_user: pi
+      ansible_password: raspberry
+      ansible_python_interpreter: python3
+
+    rasp2:
+      ansible_host: 192.168.0.24
+      ansible_user: pi
+      ansible_password: raspberry
+      ansible_python_interpreter: python3
+
+raspberry_with_camera:
+  hosts:
+    rasp2:
```

Comment:
is there a way to avoid repeating the variables for rasp2?

Reply:
Yes it seems that we can just specify the hostname `rasp2` without repearing the variables and it works.